hops/featurestore_impl/rest/rest_rpc.py

```diff
@@ -6,6 +6,30 @@
 from hops.exceptions import RestAPIError
 import json
 
+# To be compatible with python2 and python3
```

Comment:
Are we still Python 2 compatible?

Reply:
No, I copied this part from hopsworks-cloud-sdk, I can probably remove it.

Reply:
done